lib/song.rb

```diff
@@ -0,0 +1,58 @@
+class Song
+  extend Concerns::Findable
+
+  attr_accessor :name
+  attr_reader :artist
+  attr_reader :genre
```

Comment:
#### Code Quality - Nitpick
All the attr_readers should be put on the same line, separated by commas.